`yarn rw check`  fails to correctly consume `redwood.toml` contents

created per @thedavidprice request on Discord 

```yarn rw check
yarn run v1.22.10
$ /==/node_modules/.bin/rw check
(...)
web/src/index.js:12:11: warning:
This env variable is present in '.env',
but it won't be available to your app in production *unless* you add it to includeEnvironmentVariables.
Tip: add the following to your redwood.toml:

[web]
  includeEnvironmentVariables = ['AUTH0_DOMAIN']

(...)

:sparkles:  Done in 1.47s.
```
  `redwood.toml` has
```
[web]
  port = 8910
  apiProxyPath = "/.redwood/functions"
  includeEnvironmentVariables = [
    'AUTH0_DOMAIN',
    'AUTH0_CLIENT_ID',
    'AUTH0_AUDIENCE',
    'DEPLOY_URL'
  ]
```

Thanks in advance, 

All the best

--- a/packages/structure/src/model/RWEnvHelper.ts
+++ b/packages/structure/src/model/RWEnvHelper.ts
@@ -74,7 +74,7 @@ export class RWEnvHelper extends BaseNode {
   private env_default_merged_filtered(include: string[]): EnvVarMap {
     return pickBy(
       this.env_default_merged,
-      (v, k) => k.startsWith('REDWOOD_ENV_') || include?.includes(k)
+      (_v, k) => k.startsWith('REDWOOD_ENV_') || include?.includes(k)
     )
   }
 

--- a/packages/structure/src/model/RWEnvHelper.ts
+++ b/packages/structure/src/model/RWEnvHelper.ts
@@ -74,7 +74,7 @@ export class RWEnvHelper extends BaseNode {
   private env_default_merged_filtered(include: string[]): EnvVarMap {
     return pickBy(
       this.env_default_merged,
-      (k) => k.startsWith('REDWOOD_ENV_') || include?.includes(k)
+      (v, k) => k.startsWith('REDWOOD_ENV_') || include?.includes(k)
     )
   }
 

